js/layout-javascript/simple-list-code.js

```diff
@@ -2347,6 +2347,7 @@ DynamicList.prototype.saveComment = function(entryId, commentId, value) {
       return instance.update({
         settings: commentData.data.settings
       }, {
+        id: commentId,
```

Comment:
The `where` clause  beneath would be unnecessary if we pass the `id` property.

Reply:
fixed